src/components/AuthenticationModal/AuthenticationQR.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from 'react'
+import { BallTriangle } from 'react-loader-spinner'
+import {
+  AuthorizationResponseStateStatus,
+  AuthStatusResponse,
+  GenerateAuthRequestURIResponse
+} from './auth-model'
+import {
+  CreateElementArgs,
+  QRType,
+  URIData,
+  ValueResult
+} from '@sphereon/ssi-sdk-qr-react'
```

Comment:
Should be latest QR package from the SSI-SDK, where the name of the package is different from ssi-sdk-qr-react

Reply:
Fixed!